src/util/voxel_io.hpp

```diff
@@ -13,11 +17,15 @@ namespace voxel_utility {
 class VoxelObject {
  private:
     std::vector<uint32_t> data_;
-    std::vector<int> center_;
-    std::vector<uint32_t> size_;
+    std::array<int32_t, 3> center_;
+    std::array<uint32_t, 3> size_;
     bool ok_;
 
-    inline int get_position(int x, int y, int z) const {
+    quill::Logger* logger = logging::get_logger("voxel_io");
```

Comment:
1 shoulden't this be static?
2 I liked it better as util.voxel_io

Reply:
1. No because static class vars are created before `main()` runs and we need it to run after (as we init loggers in main).
2. We know voxel io is in utils, not needed

Reply:
IDK it seems to work fine
`2022-12-18T23:12:47.180 -0500 [19244 ] [voxel_io.cpp:125  ] DEBUG     [voxel_io      ] - Voxel grid size: 5 x 5 x 12`